terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -27,7 +27,7 @@
 
       it "raises a helpful error" do
         expect { subject }.to raise_error(Dependabot::DependencyFileNotEvaluatable) do |boom|
-          expect(boom.message).to eq("Invalid registry source specified: 'consul/aws'")
+          expect(boom.message).to eq("Invalid registry source specified: 'invalid/source/hashicorp/consul/aws'")
```

Comment:
I think we'll still want to make sure that a module with `consul/aws` doesn't parse as it's not valid. I think we can safely revert this